python/mxnet/initializer.py

```diff
@@ -431,8 +432,32 @@ def _init_weight(self, _, arr):
 
 
 @register
+class LSTMCellForgetBias(Initializer):
```

Comment:
The name "LSTMBias" means that we won't add weight initialization logics in it. Also the naming convention would be slightly different than FusedRNN(Initializer).